eqcorrscan/tests/match_filter_test.py

```diff
@@ -586,6 +586,19 @@ def test_tribe_copy(self):
             self.assertEqual(t, copy_t)
         self.assertEqual(tribe, copied)
 
+    def test_tribe_copy_quick(self):
```

Comment:
This looks like the wrong test for this PR.

Reply:
Ups.. thanks for removing!